homeassistant/components/dsmr/sensor.py

```diff
@@ -349,6 +349,24 @@ class DSMRSensorEntityDescription(
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    DSMRSensorEntityDescription(
+        key="belgium_current_average_demand",
+        translation_key="current_average_demand",
+        obis_reference=obis_references.BELGIUM_CURRENT_AVERAGE_DEMAND,
+        dsmr_versions={"5B"},
+        force_update=True,
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
This is an averaged value, and thus cannot have a state class.

Reply:
Same for this, value comes straight from a DSMR Telegram, and shows your max peak usage for this current month

Reply:
I get where it is coming from, but state classes are only assigned to current values, not averages & aggregations.

Where it is coming from, is not relevant for that.

Reply:
Fixed